packages/flutter/lib/src/services/binding.dart

```diff
@@ -204,6 +205,13 @@ mixin ServicesBinding on BindingBase, SchedulerBinding {
     }
     return null;
   }
+
+  /// The [RestorationManager] synchronizes the restoration data between
+  /// engine and framework.
+  ///
+  /// See the docs for [RestorationManager] for a discussion of restoration
+  /// state and how it is organized in Flutter.
+  final RestorationManager restorationManager = RestorationManager();
```

Comment:
do we want to make this be created by a virtual method so that it can be easily overriden?

Reply:
Good idea. Added `createRestorationManager` to parallel `createBinaryMessenger`.